Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'vite_rails'
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
-  gem 'capybara'
+  gem 'capybara-selenium'
```

Comment:
I've never used capybara-selenium before, and I can confirm that you don't need it for capybara to make use of selenium webdriver.

If you have a reason you've added it, feel free to keep it but if it's to make the setup work then it's unneeded and can be removed.